src/libmongoc/tests/json/client_side_encryption/explain.json

```diff
@@ -123,6 +123,9 @@
         "autoEncryptOpts": {
           "kmsProviders": {
             "aws": {}
+          },
+          "extra": {
+            "__csfleDisabled": true
```

Comment:
Confirming, has the spec test this test file corresponds to been updated in the spec repo? If not, this test might not belong under the `json` directory.

Reply:
True. This is somewhat experimenting on how to "disable" csfle for testing. We don't want to publish an "official" way to disable csfle, but this spec test also changed behavior depending on whether csfle is present. It's a catch-22.

Reply:
I think it is fine to include the field in spec tests so long as it is clearly documented as for testing purposes only. However, if the specification repo will not updated to match proposed test file contents, I would like the `__csfleDisabled` test to be moved out of the `json` directory and tested as a prose test instead.